schema/schema.yaml

```diff
@@ -254,6 +258,41 @@ tables:
         - "#Publications.reference"
 
 
+  - name: ParametersList
```

Comment:
Do we want this to be `ParameterList` vs `ParametersList` (single or plural Parameter). I believe we have AssociationList (single) so for consistency we should pick one or the other.